apps/git-pulse/src/controller/getSearchSuggestions.ts

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+
+const token = process.env.GITHUB_ACCESS_TOKEN;
+const query = 'angular'; //Provisional query.
+
+const getSearchSuggestions = async () => {
+  try {
+    const response = await axios.get(`https://api.github.com/search/repositories?q=${query}&per_page=10`, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+
```

Comment:
Please create a new file in src/services/githubService.js and put in there all the axios/api logic to allow to get data from the github api. This will help us to separate the responsabilities (remember the SOLID principles).